plugins/woocommerce-admin/client/analytics/components/report-filters/test/index.js

```diff
@@ -0,0 +1,52 @@
+/**
+ * External dependencies
+ */
+import { render } from '@testing-library/react';
+import { recordEvent } from '@woocommerce/tracks';
+
+import userEvent from '@testing-library/user-event';
+
+/**
+ * Internal dependencies
+ */
+import ReportFilters from '..';
+
+jest.mock( '@woocommerce/tracks', () => ( { recordEvent: jest.fn() } ) );
+
+describe( 'ReportFilters', () => {
+	test( 'should fire analytics_filter when filter is changed ', async () => {
```

Comment:
it would be good to make it more clear that this is testing Tracks events...

Maybe something like `should record analytics_filter Tracks event when filter is changed`?